src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -234,6 +234,7 @@ const useTravelAidAnnouncements = (
 ) => {
   const {language, t} = useTranslation();
   const isFirstRender = useRef(true);
+  const previousQuayId = useRef(state.focusedEstimatedCall.quay.id);
```

Comment:
Could the hook `useStablePreviousValue` be used for this? Also maybe the `isFirstRender` part can be removed? Since a previous quay id of `undefined` means that it is first render.

Reply:
I don't immediately see how `useStablePreviousValue` can be used here, since stablePreviousValue uses useState internally, which I don't think will work as intended in our useEffect.

The isFirstRender could indeed be removed 👍

Reply:
Aha, I didn't look through it properly. Other `usePrevious` hooks that I have seen/used have used refs, and I thought this was a variation of that.